### ISSUE TITLE
Problems Running in CI / Docker for Latest?

### ISSUE THREAD

**Issue by steve-gray** [yoheimuta/protolint#174]
Am using Drone CI, and I'm finding that as of the last few days the protolint stage in our repos is failing. No error message, the pod just suddenly terminates. Its as if the entry/command isn't running for the container.....

![image](https://user-images.githubusercontent.com/14563775/133854818-e8bb6e3f-4167-41b1-b831-26a372df7813.png)

The issue seems to be as of v0.33.0 onward. Reverting to 0.32.0 is fine. The CI block being used in drone is fairly straightforward. The below works for 0.32, but not 0.33 onward:

```
- name: proto_lint
  image: yoheimuta/protolint:v0.32.0
  commands:
    - protolint ./proto/*.proto
  depends_on: []
```

Has the way the command needs to be invoked changed or something else about the container entry point changed?

**Comment by yoheimuta** [comment]
@steve-gray Thank you for reaching out!

> something else about the container entry point changed?

Although there's no entry point changed,

```
(base) ~/w/p/g/s/g/y/protolint ❯❯❯ docker run yoheimuta/protolint:v0.32.0 version
protolint version v0.32.0(3b9f029)
(base) ~/w/p/g/s/g/y/protolint ❯❯❯ docker run yoheimuta/protolint:v0.33.0 version
protolint version 0.33.0(1b370c6)
```

v0.33.0 has a change on Dockerfile.

- GitHub: https://github.com/yoheimuta/protolint/compare/v0.32.0...v0.33.0#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R1
- DockerHub: https://hub.docker.com/layers/152840460/yoheimuta/protolint/v0.32.0/images/sha256-01798f36df3a0e45c7732218c029031cbcf29eb3c173e677ffca65a9fa2bdb71?context=repo and https://hub.docker.com/layers/166102115/yoheimuta/protolint/v0.33.0/images/sha256-8c4163c8f30d298f849dbe9bdebbed69c40ef3f51f185092e37d571dfc6cb858?context=repo


It seems problematic to put a binary in a different directory.
I'm going to revert this change on the binary location so that the executable will be found in $PATH.

```
(base) ~/w/p/g/s/g/y/protolint ❯❯❯ docker run --entrypoint "protolint" yoheimuta/protolint:v0.32.0

Protocol Buffer Linter Command.

Usage:  
        protolint <command> [arguments]

The commands are:
        lint     lint protocol buffer files
        list     list all current lint rules being used
        version  print protolint version
(base) ~/w/p/g/s/g/y/protolint ❯❯❯ docker run --entrypoint "protolint" yoheimuta/protolint:v0.33.0
docker: Error response from daemon: OCI runtime create failed: container_linux.go:380: starting container process caused: exec: "protolint": executable file not found in $PATH: unknown.
```

**Comment by yoheimuta** [comment]
@steve-gray I fixed it by v0.34.1.
Can you confirm this?

```
❯❯❯ docker run --entrypoint "protolint" yoheimuta/protolint:v0.34.1
Unable to find image 'yoheimuta/protolint:v0.34.1' locally
v0.34.1: Pulling from yoheimuta/protolint
a8d7e13e17f6: Pull complete 
Digest: sha256:af96656162f1839e46069cf66948a489e0a2a795ae9245fc242df7c690b93667
Status: Downloaded newer image for yoheimuta/protolint:v0.34.1

Protocol Buffer Linter Command.

Usage:
        protolint <command> [arguments]

The commands are:
        lint     lint protocol buffer files
        list     list all current lint rules being used
        version  print protolint version
```

**Comment by steve-gray** [comment]
Boom, fixed. Thanks @yoheimuta  - I'll unpin the projects I'd set to 0.32 now. Thank-you for the fast turnaround!

![image](https://user-images.githubusercontent.com/14563775/133868428-e2e38b5c-2cae-4bcb-a6c6-81d40cd414dc.png)
